airbyte-integrations/connectors/source-mixpanel/unit_tests/unit_test.py

```diff
@@ -1,12 +1,63 @@
 #
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
-
+import logging
 from datetime import date, timedelta
 
 from airbyte_cdk.sources.streams.http.auth import NoAuth
 from source_mixpanel.source import Annotations
+from source_mixpanel.source import SourceMixpanel
+
+logger = logging.getLogger("test_client")
+
+def test_check_connection_api_secret_ok(requests_mock):
+
+    config = {}
+    config["api_secret"] = "testApiSecret"
+
+    service_account_headers = {
+        "Authorization": "Basic api_secret:testApiSecret",
+        "Accept": "application/json",
+    }
+
+    requests_mock.register_uri("GET", "https://mixpanel.com/api/2.0/funnels/list", headers=service_account_headers)
+    ok, error_msg = SourceMixpanel().check_connection(logger, config=config)
+
+    assert ok
+    assert not error_msg
+
+def test_check_connection_service_account_ok(requests_mock):
+
+    config = {}
+    config["serviceaccount_username"] = "testName"
+    config["serviceaccount_secret"] = "testSecretName"
```

Comment:
You can do `config = {"serviceaccount_username"] = "testName", "serviceaccount_secret"] = "testSecretName"}` here.

Reply:
done